src/components/radio/radio.spec.ts

```diff
@@ -393,6 +401,20 @@ describe('MdRadio', () => {
       expect(weatherRadioInstances[1].checked).toBe(false);
       expect(weatherRadioInstances[2].checked).toBe(true);
     });
+
+    it('should add correct aria-label when actionName is defined', () => {
+      let nativeRadioInput = <HTMLElement> fruitRadioNativeElements[0].querySelector('input');
+
+      fixture.detectChanges();
+      expect(nativeRadioInput.getAttribute('aria-label')).toBe('Banana');
+    });
+
+    it('should not add aria-label when actionName is not provided', () => {
+      let nativeRadioInput = <HTMLElement> fruitRadioNativeElements[1].querySelector('input');
+
+      fixture.detectChanges();
+      expect(nativeRadioInput.getAttribute('aria-label')).toBe(null);
```

Comment:
I would also test the `aria-label` and `aria-labelled-by` attribute as a bound property, which means modifying the `@Input`'s input at runtime triggering a change detection cycle.